tcms/testruns/tests/test_admin.py

```diff
@@ -63,6 +64,13 @@ def setUpTestData(cls):
         super().setUpTestData()
         initiate_user_with_default_setups(cls.tester)
 
+    positive = TestExecutionStatus.objects.all().filter(weight__gt=0)
+    positive_list = list(positive)
```

Comment:
No need for extra intermediary variables. Python is a dynamically typed language which means you can convert the queryset to a list and assign to the same variable like before. By doing so the `positive` variable will change its type after the assignment.